src/client/Microsoft.Identity.Client/Prompt.cs

```diff
@@ -38,12 +38,18 @@ public struct Prompt
         /// <remarks>This is the recommended prompt</remarks>
         public static readonly Prompt NoPrompt = new Prompt("no_prompt");
 
+        /// <summary>
+        /// AcquireToken will send <c>prompt=create</c> to Azure AD's authorize endpoint
+        /// which would trigger a sign-up experience.
```

Comment:
```suggestion
        /// which would trigger a sign-up experience, used for External Identities. 
```

do we have an aka.ms link for this? https://docs.microsoft.com/en-us/azure/active-directory/external-identities/self-service-sign-up-user-flow @jmprieur

Reply:
here: https://aka.ms/msal-net-prompt-create @jmprieur @pmaytak